idea_town/frontend/static-src/.editorconfig

```diff
@@ -0,0 +1,12 @@
+# http://editorconfig.org
```

Comment:
Yeah, I've got filetype-based rules set up in vim, but that's awfully neckbeardy to expect of community members.

If you want to keep this PR going even longer, feel free to move this file into the root. Otherwise, might be worth it to file a bug.